rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -851,14 +865,14 @@ private static Stream<Arguments> unionResponseCodeProvider() {
         }
 
         @Test
-        void getUnionBridgeLockingCap_beforeRskip502_shouldReturnNegativeOne() {
+        void getUnionBridgeLockingCap_beforeRskip502_shouldReturnEmpty() {
```

Comment:
empty? I thought it was no longer Optional

Reply:
Renamed